tools/eval.py

```diff
@@ -56,8 +56,8 @@ def main():
     sys.stdout = Logger(osp.join(cfg.data.save_dir, log_name))
 
     datamanager = torchreid.data.ImageDataManager(filter_classes=args.classes, **imagedata_kwargs(cfg))
-    num_classes = len(datamanager.test_loader[cfg.data.targets[0]]['query'].dataset.classes)
-
+    num_classes = datamanager.test_loader[cfg.data.targets[0]]['query'].dataset.num_train_pids[0]
```

Comment:
This looks like a hack. Please fix `MultiLabelClassification` to have a valid list of classes here.

Reply:
Yes, I added this to my check box to-do list

Reply:
done